cql3/restrictions/statement_restrictions.cc

```diff
@@ -969,5 +975,229 @@ bool single_column_restriction::LIKE::is_satisfied_by(bytes_view data, const que
     return (*_matcher)(data);
 }
 
+namespace wip {
+
+namespace {
+
+using cql3::selection::selection;
+
+/// Returns cdef's value from the fetched data.
+bytes_opt get_value(const column_definition* cdef, const selection& selection,
+                    const std::vector<bytes>& partition_key, const std::vector<bytes>& clustering_key,
+                    const std::vector<bytes_opt>& other_columns) {
```

Comment:
Perhaps (later) we can provide the last three parameters as row_data.

Reply:
You don't mean `repair_stream_cmd::row_data`, I presume, but a new, aggregate class?

Reply:
Yes, just for cosmetic reasons.

Reply:
Done.